src/util/simd_readme.md

```diff
@@ -0,0 +1,332 @@
+# SIMD classes for Arbor
+
```

Comment:
A lot of the content in here is repeated in the html docs.
To avoid maintaining two readme/documentations, how about moving the extra information here into the RTD docs?